packages/kotti-ui/tsconfig.json

```diff
@@ -12,7 +12,6 @@
 		"strict": true,
 		"jsx": "react",
 		"jsxFactory": "h",
-		// FIXME: remove skipLibCheck when [https://github.com/ElemeFE/element/issues/17329] is resolved
```

Comment:
@smeijer this is perhaps something that could be fixed in `unimported`. It doesn’t seem to support comments in JSON. I know, not technically allowed in the JSON standard, however neither rollup, typescript, vscode, etc. had any issues with this. Removed it for `unimported` for now as it’s not a big deal in this particular case

Reply:
Thanks. I'll fix that in a future release. It's being tracked via this issue. https://github.com/smeijer/unimported/issues/49#issuecomment-894072532